Sources/Common/Core/Math/test/testMath.js

```diff
@@ -754,3 +754,47 @@ test('Test singularValueDecomposition', (t) => {
   }
   t.end();
 });
+
+test.only('Test getSparseOrthogonalMatrix', (t) => {
+  const successMsg = 'Row/column association is correct';
+  const tests = [
+    {
+      matrix: [
+        0.702, 0.7025, -0.1163, -0.2856, 0.1281, -0.9497, -0.6523, 0.7, 0.2906,
+      ],
+      expected: [0, 1, 0, 0, 0, -1, -1, 0, 0],
+    },
```

Comment:
can you maybe also add as a test the problem you faced with Kitware Glance ?

Reply:
It is the first test matrix, which use to associate both X and Z to J